recipes/crocoddyl/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "crocoddyl" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/loco-3d/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ef2cc3e7ccf221900df20c6513f01ee30b49c55aa2d00421b5dedcc0cf061266
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - clang  # [win]
+    - make
+  host:
+    - eigen
+    - eigenpy
+    - pinocchio
+    - boost
+    - python
+    - example-robot-data
```

Comment:
Is the example data really needed at build time?

Reply:
Fixed.

Reply:
It seems, based on the failed builds, that `example-robot-data` really *is* needed at build time, so probably need to add that back.